README.md

```diff
@@ -1,10 +1,15 @@
-## ⛅️ wrangler
+## ⛅️ Home to `wrangler`, the CLI for Cloudflare Workers®, as well as other tools for interacting with Workers
```

Comment:
@IgorMinar thoughts?

```suggestion
## ⛅️ Home to `wrangler`, the CLI for Cloudflare Workers®, as well as other tools for interacting with Workers Developer Platform
```

Reply:
Cloudflare Developer Platform

Reply:
Nvm this isn't something to consider yet. It can easily be updated as it becomes relevant again. 😄